--- vidix/vidix/drivers/mach64_vid.c.orig	2004-01-10 19:17:33.000000000 +0100
+++ vidix/vidix/drivers/mach64_vid.c	2004-02-21 15:42:42.731488952 +0100
@@ -660,7 +660,8 @@
 	    But: (720*720*2(YUV422)*16(sizeof(bm_descriptor)))/4096=4050
 	    Thus one 4K page is far enough to describe max movie size.
 	*/
-	for(i=0;i<64;i++)
+	int i = 0;
+	for(;i<64;i++)
 	    if((mach64_dma_desc_base[i] = memalign(4096,mach64_ram_size*sizeof(bm_list_descriptor)/4096)) == 0)
 		goto out_mem;
 #if 0
@@ -691,7 +692,8 @@
   bm_close();
   if(can_use_irq && irq_installed) hwirq_uninstall(pci_info.bus,pci_info.card,pci_info.func);
   if(dma_phys_addrs) free(dma_phys_addrs);
-  for(i=0;i<64;i++) 
+  int i = 0;
+  for(;i<64;i++) 
   {
     if(mach64_dma_desc_base[i]) free(mach64_dma_desc_base[i]);
   }
