--- vidix-1.0.0/libdha/Makefile.orig	2007-04-07 08:08:23.000000000 +0200
+++ vidix-1.0.0/libdha/Makefile	2007-04-19 19:30:24.981089236 +0200
@@ -82,15 +82,14 @@
 	$(CC) -MM $(CFLAGS) $(SRCS) 1>.depend
 
 install:
-	-mkdir -p $(prefix)/lib
+	-mkdir -p $(DESTDIR)$(libdir)
 ifeq ($(VIDIX_BUILD_SHARED),yes)
-	install -D -m 755 -s -p $(LIBNAME_SHARED_VERSION) $(prefix)/lib/$(LIBNAME_SHARED_VERSION)
-	ln -sf $(LIBNAME_SHARED_VERSION) $(prefix)/lib/$(LIBNAME_SHARED)
-	ln -sf $(LIBNAME_SHARED_VERSION) $(prefix)/lib/$(LIBNAME_SHARED_MAJOR)
-	-$(LDCONFIG) $(LDCONFIG_FLAGS)
+	install -D -m 755 -p $(LIBNAME_SHARED_VERSION) $(DESTDIR)$(libdir)/$(LIBNAME_SHARED_VERSION)
+	ln -sf $(LIBNAME_SHARED_VERSION) $(DESTDIR)$(libdir)/$(LIBNAME_SHARED)
+	ln -sf $(LIBNAME_SHARED_VERSION) $(DESTDIR)$(libdir)/$(LIBNAME_SHARED_MAJOR)
 endif
 ifeq ($(VIDIX_BUILD_STATIC),yes)
-	cp -f $(LIBNAME_STATIC) $(prefix)/lib/$(LIBNAME_STATIC)
+	cp -f $(LIBNAME_STATIC) $(DESTDIR)$(libdir)/$(LIBNAME_STATIC)
 endif
 
 uninstall:
--- vidix-1.0.0/vidix/Makefile.orig	2007-04-07 08:08:23.000000000 +0200
+++ vidix-1.0.0/vidix/Makefile	2007-04-19 19:31:41.453447147 +0200
@@ -8,7 +8,7 @@
 LIBNAME_STATIC         = $(NAME).a
 LIBNAME_WIN32          = $(NAME).dll
 
-LIBDIR = $(prefix)/lib
+LIBDIR = $(libdir)
 INCDIR = $(prefix)/include/vidix
 
 SRCS        = vidixlib.c
@@ -68,21 +68,20 @@
 	$(MAKE) -C drivers depend
 
 install:
-	mkdir -p $(LIBDIR)
+	mkdir -p $(DESTDIR)$(LIBDIR)
 ifeq ($(VIDIX_BUILD_SHARED),yes)
-	install -m 755 -s -p $(LIBNAME_SHARED_VERSION) $(LIBDIR)
-	ln -sf $(LIBNAME_SHARED_VERSION) $(prefix)/lib/$(LIBNAME_SHARED)
-	ln -sf $(LIBNAME_SHARED_VERSION) $(prefix)/lib/$(LIBNAME_SHARED_MAJOR)
-	-$(LDCONFIG) $(LDCONFIG_FLAGS)
+	install -m 755 -p $(LIBNAME_SHARED_VERSION) $(DESTDIR)$(LIBDIR)
+	ln -sf $(LIBNAME_SHARED_VERSION) $(DESTDIR)$(LIBDIR)/$(LIBNAME_SHARED)
+	ln -sf $(LIBNAME_SHARED_VERSION) $(DESTDIR)$(LIBDIR)/$(LIBNAME_SHARED_MAJOR)
 endif
 ifeq ($(VIDIX_BUILD_STATIC),yes)
-	cp -f $(LIBNAME_STATIC) $(prefix)/lib
+	cp -f $(LIBNAME_STATIC) $(DESTDIR)$(LIBDIR)
 endif
 
-	mkdir -p $(INCDIR)
-	install -m 644 fourcc.h $(INCDIR)
-	install -m 644 vidix.h $(INCDIR)
-	install -m 644 vidixlib.h $(INCDIR)
+	mkdir -p $(DESTDIR)$(INCDIR)
+	install -m 644 fourcc.h $(DESTDIR)$(INCDIR)
+	install -m 644 vidix.h $(DESTDIR)$(INCDIR)
+	install -m 644 vidixlib.h $(DESTDIR)$(INCDIR)
 
 	$(MAKE) -C drivers install
 
--- vidix-1.0.0/vidix/drivers/Makefile.orig	2007-04-05 09:24:34.000000000 +0200
+++ vidix-1.0.0/vidix/drivers/Makefile	2007-04-19 19:32:20.899695061 +0200
@@ -1,7 +1,7 @@
 include ../../config.mak
 
-LIBDIR = $(prefix)/lib/vidix
-OLDLIBDIR = $(prefix)/lib/mplayerxp/vidix
+LIBDIR = $(libdir)/vidix
+OLDLIBDIR = $(libdir)/mplayerxp/vidix
 
 VIDIX_LIBS = -L../../libdha -ldha
 VIDIX_LIBS_M = $(VIDIX_LIBS) -lm
@@ -99,9 +99,8 @@
 	$(CC) -MM $(CFLAGS) $(SRCS) 1>.depend
 
 install:
-	mkdir -p $(LIBDIR)
-	install -m 755 -p *.so $(LIBDIR)
-	if test -d $(OLDLIBDIR) ; then rm -f $(OLDLIBDIR)/*.so ; rmdir -p --ignore-fail-on-non-empty $(OLDLIBDIR) ; fi
+	mkdir -p $(DESTDIR)$(LIBDIR)
+	install -m 755 -p *.so $(DESTDIR)$(LIBDIR)
 
 uninstall:
 	rm -f $(LIBDIR)/*.so
