--- vidix-0.9.9.2/libdha/Makefile.orig	2006-12-13 09:25:13.049505000 +0100
+++ vidix-0.9.9.2/libdha/Makefile	2006-12-13 09:26:17.259505000 +0100
@@ -34,7 +34,7 @@
 	$(CC) -c $(CFLAGS) -o $@ $<
 
 $(LIBNAME):     $(OBJS)
-	$(CC) -shared -o $(LIBNAME) $(OBJS) $(LIBS)
+	$(CC) -shared -Wl,-soname=$(LIBNAME) -o $(LIBNAME) $(OBJS) $(LIBS)
 ifneq  ($(TARGET_OS),CYGWIN)
 	ln -sf $(LIBNAME) $(SHORTNAME)
 endif
