diff -uNr vidix/libdha/Makefile vidix.built/libdha/Makefile
--- vidix/libdha/Makefile	2003-02-11 17:04:35.000000000 +0100
+++ vidix.built/libdha/Makefile	2004-02-21 15:46:31.000000000 +0100
@@ -54,9 +54,9 @@
 	$(CC) -MM $(CFLAGS) $(SRCS) 1>.depend
 
 install:
-	install -D -m 755 -s -p $(LIBNAME) $(prefix)/lib/$(LIBNAME)
-	rm -f $(prefix)/lib/libdha.so
-	ln -sf $(LIBNAME) $(prefix)/lib/libdha.so
+	install -D -m 755 -p $(LIBNAME) $(DESTDIR)$(libdir)/$(LIBNAME)
+	rm -f $(DESTDIR)$(libdir)/libdha.so
+	ln -sf $(LIBNAME) $(DESTDIR)$(libdir)/libdha.so
 ifeq ($(TARGET_OS),OpenBSD)
 	$(LDCONFIG) -R
 else
--- vidix-0.9.9.2/vidix/drivers/Makefile.orig	2005-04-27 08:09:49.000000000 +0200
+++ vidix-0.9.9.2/vidix/drivers/Makefile	2006-11-19 00:28:04.646229750 +0100
@@ -2,8 +2,8 @@
 
 include ../../config.mak
 
-BINDIR = $(prefix)/lib/vidix
-OLDBINDIR = $(prefix)/lib/mplayerxp/vidix
+BINDIR = $(DESTDIR)$(LIBDIR)
+OLDBINDIR = $(DESTDIR)$(libdir)/mplayerxp/vidix
 ifeq ($(TARGET_OS),CYGWIN)
 L_FLAGS=
 LINKER=$(CC)
--- vidix-0.9.9.2/vidix/Makefile.orig	2005-06-28 17:20:32.000000000 +0200
+++ vidix-0.9.9.2/vidix/Makefile	2006-11-19 00:28:24.303458250 +0100
@@ -5,8 +5,8 @@
 else
 LIBNAME = libvidix.so
 endif
-BINDIR = $(prefix)/lib
-INCDIR = $(prefix)/include/vidix
+BINDIR = $(DESTDIR)$(libdir)
+INCDIR = $(DESTDIR)$(prefix)/include/vidix
 SUBDIRS = drivers
 DO_MAKE = @ for i in $(SUBDIRS); do $(MAKE) -C $$i $@ || exit; done
 
